### PR TITLE
Replace $GLOBALS['I18N']->get() with s()

### PR DESCRIPTION
**Pull request by samtuke**
Switch to newer translation function

### CODE REVIEW

File: public_html/lists/admin/mviews.php [hunk 1]
```diff
@@ -84,10 +84,10 @@
 
 $total = Sql_Affected_Rows();
 if (isset($start) && $start > 0) {
-    $listing = sprintf($GLOBALS['I18N']->get('Listing user %d to %d'), $start, $start + MAX_USER_PP);
+    $listing = sprintf(s('Listing user %d to %d'), $start, $start + MAX_USER_PP);
```
Comment (michield) [inline public_html/lists/admin/mviews.php]: the s function can handle %d so this one could become

$listing = s('Listing user %d to %d', $start, $start + MAX_USER_PP);

Comment (samtuke) [reply public_html/lists/admin/mviews.php]: Great, added

**Comment by samtuke** [comment]
Rebased on origin master; ready to merge

**Comment by michield** [comment]
can't merge this, there are conflicts

**Pull request merged**